vue.config.js

```diff
@@ -25,7 +26,8 @@ module.exports = {
 								'x64'
 							]
 						}
-					]
+					],
+					icon: 'build/256x256.png'
```

Comment:
Apart from this one, macOS actually wants a 512x512 lmao

Reply:
Seems like the merge from upstream changed it back, I'll change it back this weekend when I try to finish this PR :eyes: